cxx-qt-gen/src/gen_cpp.rs

```diff
@@ -1039,7 +1039,7 @@ pub fn generate_qobject_cpp(obj: &QObject) -> Result<CppObject, TokenStream> {
     // Generate C++ source part
     let source = formatdoc! {r#"
         #include "cxx-qt-gen/include/{ident_snake}.h"
-        #include "cxx-qt-gen/src/{ident_snake}.rs.h"
+        #include "cxx-qt-gen/include/{ident_snake}_cxx.h"
```

Comment:
won't the folder on both of these includes now go ? As in the CMake we'll add the `include/` folder to the include directories ? and the output won't necessarily be `cxx-qt-gen/include/` but it could be like `build/rust/include` or anything etc.

Reply:
This then makes me wonder what should happen with `#include "cxx-qt-lib/include/qt_types.h"` maybe when we have the tool output the cxx-qt-lib stuff into a folder we can have it produce `cxx-qt-lib/qt_types.h` so then the include becomes like `#include <cxx-qt-lib/qt_types.h>` or something.

Reply:
As discussed, lets figure this out in later changes.